packages/styles/accordion.css

```diff
@@ -75,3 +75,10 @@ button.Accordion__trigger {
   border-radius: 0 0 3px 3px;
   box-shadow: 0 2px 3px 0 var(--accordion-box-shadow-color);
 }
+
+@media (prefers-reduced-motion: reduce) {
+  .Accordion__trigger:hover,
+  .Accordion .Icon {
+    transition: none;
+  }
+}
```

Comment:
An alternative to using this media query would be to add/remove a class from the Accordion. However, using the CSS media query seems safer/cleaner to use here.

Reply:
I'm not sure we need `prefers-reduced-motion` here. Setting this value doesn't mean "no motion", but removing non-essential motion. I would argue that the slight motion that happens here (the rotation of caret and the highlight of the trigger) are still subtile enough where it isn't needed.

Reply:
From a user experience perspective, the accordion animation can also be helpful for users to notice the change of content and what is occurring on the screen.